src/main/resources/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep/help.html

```diff
@@ -1,14 +1,69 @@
 <div>
     <p>
-        Takes a map from branch names to closures and an optional argument <code>failFast</code>
-        which will terminate all branches upon a failure in any other branch:
-    </p>
-    <pre>
-    parallel firstBranch: {
-        // do something
-    }, secondBranch: {
-        // do something else
-    },
-    failFast: true|false
-    </pre>
+      Takes a map from branch names to closures and runs the closures in
+      parallel. The optional <code>failFast</code> option terminates
+      all jobs as soon as any job fails.
```

Comment:
Would suggest avoiding the term _job_ in this context as for Jenkins this has the specific meaning of `hudson.model.Job`, a.k.a. _project_. Oh, I hear the voice of @bitwiseman in my head telling me to talk less and suggest more, so

```suggestion
      all branches as soon as any branch fails.
```